README.md

```diff
@@ -1,35 +1,35 @@
 # Node.js tools for Visual Studio
 
-Node.js tools for Visual Studio 2017 is developed and managed here.
+Node.js tools for Visual Studio is developed and managed here.
 
 ![Build status for master branch](https://devdiv.visualstudio.com/_apis/public/build/definitions/0bdbc590-a062-4c3f-b0f6-9383f67865ee/4492/badge)
 [![Join the chat at https://gitter.im/Microsoft/nodejstools](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Microsoft/nodejstools?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
 
 Check out our documentation on [the NTVS wiki](https://github.com/Microsoft/nodejstools/wiki) 
-and our (old) [feature overview](https://channel9.msdn.com/events/Visual-Studio/Connect-event-2015/801) video on Channel 9.
 
 Feel free to file issues or ask questions on our [issue tracker](http://github.com/Microsoft/nodejstools/issues), 
 and we welcome code contributions - see 
 [Contributing](https://github.com/Microsoft/nodejstools/blob/master/.github/CONTRIBUTING.md) for information. 
 
-## Visual Studio 2017 installation
+## Visual Studio 2019 installation
 
-The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
+The Node.js development workload is available as part of Visual Studio 2019 Community, Professional 
 and Enterprise. 
 To install, run the [normal VS installer](https://visualstudio.com/vs/downloads?wt.mc_id=github_microsoft_com) 
 and select the **Node.js development workload**.
 
-For the latest update to the Node.js tools install [Visual Studio Preview](https://www.visualstudio.com/vs/preview/).
+## Visual Studio 2017 installation
 
-## Visual Studio 2015 and earlier
+The Node.js development workload is available as part of Visual Studio 2017 Community, Professional 
+and Enterprise. 
+To install, head to run the [2017 VS installer](https://visualstudio.microsoft.com/vs/older-downloads/). Please note you will need a Visual Studio Subscription to access this.
+and select the **Node.js development workload**.
 
-Prior to Visual Studio 2017, the Node.js tools were released as a standalone extension. We are no longer actively developing these versions, 
-but if you are unable to upgrade to Visual Studio 2017 yet, you can use these older installers to get the Node.js tools.
 
-[NTVS 1.3.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.3.1) is the latest available version for Visual Studio 2015.
+## Visual Studio 2015 and earlier
 
-[NTVS 1.1](https://github.com/Microsoft/nodejstools/releases/tag/v1.1.1) is the last release for Visual Studio 2012, and 2013.
+For more information on how to install Node.js tools, please send an email to (Gabrielle Crevecoeur (Current Program Manager))[mailto:gacreve@microsoft.com?subject=Installing%20Node.js%20Tools%20For%20Early%20Versions%20%Of%20VS]
```

Comment:
This mailto link is not rendering properly in the preview. I think you need `[]` around your name and `()` around the actual link.

Reply:
Youre totally right. I mixed up my () and []

Reply:
good catch !